sdks/python/apache_beam/typehints/schemas.py

```diff
@@ -666,6 +720,28 @@ def _from_typing(cls, typ):
     return cls()
 
 
+class PassThroughLogicalType(LogicalType[LanguageT, LanguageT, ArgT]):
+  """A base class for LogicalTypes that use the same type as the underlying
+  representation type.
+  """
+  def to_language_type(self, value):
+    return value
+
+  @classmethod
+  def representation_type(cls):
+    # type: () -> type
+    return cls.language_type()
+
+  def to_representation_type(self, value):
+    return value
+
+  @classmethod
+  def _from_typing(cls, typ):
+    # type: (type) -> LogicalType
+    # TODO: enable argument
```

Comment:
can we link an issue here (it could be the same one for following up on non-atomic types)

Reply:
Thanks, linked to #23373